src/core/public/styles/_base.scss

```diff
@@ -1,6 +1,5 @@
 // Charts themes available app-wide
 @import '@elastic/charts/dist/theme';
-@import '@elastic/eui/src/themes/charts/theme';
```

Comment:
@nickofthyme Would appreciate your help QAing this and confirming that the removed `.echAnnotation` CSS does not materially affect Kibana (see https://github.com/elastic/eui/pull/7572#pullrequestreview-1934890909)

Reply:
@cee-chen did a check around kibana and see no impact of removing these styles.